src/hotspot/cpu/aarch64/vm_version_aarch64.cpp

```diff
@@ -120,6 +120,11 @@ void VM_Version::initialize() {
     ContendedPaddingWidth = dcache_line;
   }
 
+  if (!(is_aligned(CryptoPmullForCRC32LowLimit, 128))) {
+    warning("CryptoPmullForCRC32LowLimit must be a multiple of 128");
+    CryptoPmullForCRC32LowLimit = align_down(CryptoPmullForCRC32LowLimit, 128);
+  }
```

Comment:
Can you describe somewhere why it has to be a multiple of `128`? Imagine someone comes across this later, and wonders if that is just some strange implementation limitation or something more fundamental, or something very subtle.